docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -13,42 +13,54 @@ The sample configuration files below run Semgrep CI on various continuous integr
 ## GitHub Actions
 
 ```yaml
+# Name of this GitHub Actions workflow.
 name: Semgrep
 
 on:
-  # Scan changed files in PRs, block on new issues only (existing issues ignored)
+  # Determine when you want Semgrep to scan your code.
+  # Use as many of the following options as you want.
+  # Option 1: scan changed files in PRs, block on new issues only (existing
+  # issues in the repository ignored).
+  # To run on specific types of PR states (opened, reopened, etc) or particular
+  # paths or branches, see the GitHub docs at:
+  # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
   pull_request: {}
 
-  # Scan all files on branches, block on any issues
+  # Option 2: scan all files on branches, block on any issues
   # push:
   #   branches: ["master", "main"]
 
-  # Schedule this job to run at a certain time, using cron syntax
+  # Option 3: schedule this job to run at a certain time, using cron syntax
   # Note that * is a special character in YAML so you have to quote this string
```

Comment:
```suggestion
  # The asterisk sign * is a special character in YAML so you have to quote this string
```

Reply:
@efortuna... have to share - I never checked the etymology of asterisk. But it appears that ἀστερίσκος is a little star! ([Wiki](https://en.wikipedia.org/wiki/Asterisk)) Reminds me of: https://youtu.be/ThHz9wlBeLU :D